### ISSUE TITLE
Error when importing some pack

### ISSUE THREAD

**Issue by marionf** [PrestaShop/PrestaShop#13463]
**Describe the bug**
Error when importing some pack

**To Reproduce**
1. Go to International > Localization
2. Select Malaysia, Norway, Pakistan, Paraguay or Panama
3. Click on Import

**Screenshots**
![capture d'écran_1425](https://user-images.githubusercontent.com/13449658/56298286-2c3d8880-6132-11e9-8911-772155e10e39.png)

**Additionnal information**
PrestaShop version: 1.7.6.x
PHP version: N/A

**Comment by marionf** [comment]
Can't reproduce :(

**Comment by marionf** [comment]
Reproduced with Norway, Pakistan, Paraguay, Panama, Malaysia

**Comment by marionf** [comment]
![capture d'écran_1428](https://user-images.githubusercontent.com/13449658/56301838-e6d08980-6138-11e9-9b27-1a8c0be67d24.png)

**Comment by sarahdib** [comment]
Reproduce with norway, pakistan, paraguay, panama & malaysia

**Comment by sarjon** [comment]
Hi @sarahdib and @marionf, I was able to import all packs successfully. :thinking:

**Comment by marionf** [comment]
I can reproduce it right now with some other localizations: Germany, France, Italia, Spain ...

**Comment by sarjon** [comment]
All of these are working fine for me. :confused:

**Comment by matks** [comment]
This bug looks lovely to fix ❤️

**Comment by tomlev** [comment]
@marionf or @sarahdib could you please share the content of your `ps_lang` table for a non-working shop ?

**Comment by sarjon** [comment]
> @marionf or @sarahdib could you please share the content of your ps_lang table for a non-working shop ?

I was not able to reproduce it, everything was working fine for me, so I'm not sure if my `ps_lang` would help you. :/

**Comment by Quetzacoalt91** [comment]
I reproduced it, and it happens everytime now. I'm still digging to find the cause, here is my progress so far:

![Capture d’écran du 2019-04-24 16-10-46](https://user-images.githubusercontent.com/6768917/56671697-fc9bfc80-66ac-11e9-9ec5-ba0e0ee77804.png)

![Capture d’écran du 2019-04-24 16-19-17](https://user-images.githubusercontent.com/6768917/56671705-ff96ed00-66ac-11e9-9ed2-4d383ffe1d1f.png)

FYI, the database content:

```
select * from ps_lang;
+---------+----------------------------+--------+----------+---------------+--------+------------------+---------------------+--------+
| id_lang | name                       | active | iso_code | language_code | locale | date_format_lite | date_format_full    | is_rtl |
+---------+----------------------------+--------+----------+---------------+--------+------------------+---------------------+--------+
|       1 | Fran�ais (French)          |      1 | fr       | fr            | fr-FR  | d/m/Y            | d/m/Y H:i:s         |      0 |
|       2 | ???? (Traditional Chinese) |      1 | tw       | zh-tw         | zh-TW  | Y-m-d            | Y-m-d H:i:s         |      0 |
|       3 | ?? (Simplified Chinese)    |      1 | zh       | zh-cn         | zh-CN  | Y-m-d            | Y-m-d H:i:s         |      0 |
|       4 | ??? (Japanese)             |      1 | ja       | ja-jp         | ja-JP  | Y-m-d            | Y-m-d H:i:s         |      0 |
|       5 | Bahasa melayu (Malay)      |      1 | ms       | ms-my         | ms-MY  | d MMMM yyyy      | d MMMM yyyy h:mm:ss |      0 |
|       6 | Bokm�l (Norwegian)         |      1 | no       | no-no         | no-NO  | Y-m-d            | Y-m-d H:i:s         |      0 |
|       7 | ???? (Urdu)                |      1 | ur       | ur            | ur-PK  | Y-m-d            | Y-m-d H:i:s         |      1 |
|       8 | English (English)          |      1 | en       | en-us         | en-US  | m/d/Y            | m/d/Y H:i:s         |      0 |
|       9 | ????? ??????? (Arabic)     |      1 | ar       | ar-sa         | ar-SA  | Y-m-d            | Y-m-d H:i:s         |      1 |
+---------+----------------------------+--------+----------+---------------+--------+------------------+---------------------+--------+
9 rows in set (0.00 sec)

```

**Comment by Quetzacoalt91** [comment]
Fixed by #13563
